tests/models/mamba/test_modeling_mamba.py

```diff
@@ -489,3 +500,49 @@ def test_simple_generate_cuda_kernels_big(self, device):
         output_sentence = self.tokenizer.decode(output[0].tolist())
 
         self.assertEqual(output_sentence, expected_output)
+
+    @parameterized.expand([("state-spaces/mamba-130m",)])
```

Comment:
I don't think it's necessary to add a test - we don't do this for other models. Instead we typically compare the original model's logits to our converted model's for a given output in the conversion script

Reply:
Makes sense. Removed the test and ran the command e2e and ran a spot test (https://gist.github.com/byi8220/9b44a5f6c6c2c7533704801478f1760a)

I was able to successfully convert their base 130m and 790m models without a problem, however my GPU setup runs out of memory trying for the 1.4b and 2.8b models.